opencti-platform/opencti-front/src/private/components/common/stix_core_objects/StixCoreObjectSimulationResult.jsx

```diff
@@ -348,47 +454,97 @@ const StixCoreObjectSimulationResult = ({ id, type }) => {
   const renderForm = () => {
     return (
       <>
+        <FormControl style={{ width: '100%' }}>
+          <InputLabel id="simulationType">{t_i18n('Simulation type')}</InputLabel>
+          <Select
+            labelId="simulationType"
+            value={simulationType}
+            onChange={(event) => setSimulationType(event.target.value)}
+            fullWidth
+          >
+            <MenuItem value="technical">{t_i18n('Technical (payloads)')}</MenuItem>
+            <MenuItem value="simulated" disabled={!isSimulatedEmailsAvailable}>{t_i18n('Simulated emails (generated by AI)')} <EEChip /></MenuItem>
+            <MenuItem value="mixed" disabled>{t_i18n('Mixed (both)')} <EEChip /></MenuItem>
+          </Select>
+        </FormControl>
+        {(simulationType !== 'simulated') && (
+          <>
+            {!hasAttackPatterns && (
+              <Alert
+                severity="warning"
+                variant="outlined"
+                style={{ marginTop: 2 }}
+              >
+                {t_i18n('Technical (payloads) require attack patterns in this entity.')}
+              </Alert>
+            )}
+            <FormControl style={fieldSpacingContainerStyle}>
+              <Autocomplete
+                id="simulationPlatforms"
+                multiple
+                options={platformOptions}
+                value={platformOptions.filter((platform) => platforms.includes(platform.value))}
+                onChange={(_event, newValue) => {
+                  const newSelectedValues = newValue.map((platform) => platform.value);
+                  setPlatforms(newSelectedValues);
+                }}
+                getOptionLabel={(option) => option.label}
+                renderInput={(params) => (
+                  <TextField
+                    {...params}
+                    label={t_i18n('Targeted platforms')}
+                    variant="standard"
+                    required
+                    error={platforms.length === 0}
+                    helperText={platforms.length === 0 ? t_i18n('This field should not be empty.') : ''}
```

Comment:
In a form, to check if the entered data are correct, we use a 'validator'. For instance, see feedCreationValidation in FeedCreation